ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -81,8 +81,7 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
                                         if (workflowOperation.title === 'DELETE') {
                                             navigation.goto(uiUrl(`workflows/${workflow.metadata.namespace}`));
                                         } else {
-                                            // navigation.goto does not seem to work here
-                                            document.location.href = uiUrl(`workflows/${wf.metadata.namespace}/${wf.metadata.name}`);
+                                            setName(wf.metadata.name);
```

Comment:
This is safe because no operation will change the `namespace` of the workflow we're looking at

Reply:
cool, I could not figure out how to get this working - great!